src/kOS/Suffixed/Part/PartValue.cs

```diff
@@ -61,7 +64,7 @@ private PartModuleFields GetModule(string modName)
             foreach (PartModule mod in Part.Modules)
             {
                 SafeHouse.Logger.Log(string.Format("Does \"{0}\" == \"{1}\"?", mod.moduleName.ToUpper(), modName.ToUpper()));
```

Comment:
Agreed.  I have a separate issue started for me to do a full audit of all calls to the logger.  But we can remove theses references as we go instead.